sosivio/README.md

```diff
@@ -2,15 +2,14 @@
  
 ## Overview
  
-Sosivio is a Predictive Troubleshooting platform built specifically for Kubernetes environments and applications. Sosivio provides predictive troubleshooting tools, root cause determination and instant remediation for any failures in Kubernetes environments. Instantly start observing all flavors of Kubernetes with real-time metrics, cluster health checks, and using lean AI to predict and prevent critical failure before they appear!
+Sosivio is a Predictive Troubleshooting platform built specifically for Kubernetes environments and applications. Sosivio provides predictive troubleshooting tools, root cause determination, and instant remediation for any failures in Kubernetes environments. Instantly start observing all flavors of Kubernetes with real-time metrics and cluster health checks, and use lean AI to predict and prevent critical failures before they appear.
 
-Sosivio’s Datadog integration allows users to see Sosivio Failure Alerts directly in Datadog dashboards and instantly be redirected to Sosivio UI to remediate those failures (with Sosivio Premium license). Additionally, add context around Datadog’s signals with Sosivio’s root cause determination for critical failures. 
+Sosivio’s Datadog integration allows users to see Sosivio Failure Alerts directly in Datadog dashboards and instantly be redirected to the Sosivio UI to remediate those failures (with Sosivio Premium license). You can also add context around Datadog’s signals with Sosivio’s root cause determination for critical failures. 
  
 ## Setup
 
-If you don’t have a Sosivio account setup, [create an account and sign up for a free 4-week trial][5] of Sosivio Premium directly from our website (no credit card required). After setting up the account, you are ready to start using the Sosivio Application in Datadog. 
+If you don’t have a Sosivio account setup, [create an account][5] and sign up for a free 4-week trial of Sosivio Premium directly from our website (no credit card required). At the end of the 4-week trial, your license converts to the Sosivio Community Version, which is free, forever. With the account set up, you are ready to start using the Sosivio Application in Datadog. 
```

Comment:
```suggestion
If you don’t have a Sosivio account set up, [create an account][5] and sign up for a free 4-week trial of Sosivio Premium directly from our website (no credit card required). At the end of the 4-week trial, your license converts to the Sosivio Community Version, which is free, forever. With the account set up, you are ready to start using the Sosivio Application in Datadog. 
```

Reply:
FYI - that caused a failure:
```
Run ddev validate readmes $CHANGED
Error:     readme contains non-ASCII character(s)
    | Sosivio’s Datadog integration allows users to see Sosivio Failure Alerts directly in Datadog dashboards and instantly be redirected to the Sosivio UI to remediate those failures (with Sosivio Premium license). You can also add context around Datadog’s signals with Sosivio’s root cause determination for critical failures. 
    |        ^                                                                                                                                                                                                                                                 ^                      ^           
``` 

probably due to the line breaks on your Mac/Windows machine.
fixed that - just letting you know :)